model/riscv_vlen.sail

```diff
@@ -0,0 +1,37 @@
+type elen : Int = 64
```

Comment:
I guess the plan is to only support ELEN=64 for this release but ELEN=32 implementations are likely to be common. ELEN=16 probably not.

Reply:
The ELEN domain is now extended to support 32.